pkgs/development/tools/gomarkdoc/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, gomarkdoc }:
+
+buildGoModule rec {
+  pname = "gomarkdoc";
+  version = "1.1.0";
+  src = fetchFromGitHub {
```

```suggestion
  version = "1.1.0";

  src = fetchFromGitHub {
```
small formatting thing